aptos-move/framework/aptos-stdlib/sources/simple_map.spec.move

```diff
@@ -56,12 +56,29 @@ spec aptos_std::simple_map {
         pragma verify=false;
     }
 
-    spec to_vec_pair {
-        pragma verify=false;
+    spec to_vec_pair<Key: store, Value: store>(map: SimpleMap<Key, Value>): (vector<Key>, vector<Value>) {
```

Comment:
Unrelated to this PR, but a remark to the author of `to_vec_pair`:

I think a better function signature should be `to_pair_vec`, i.e.,
```rust
fn to_pair_vec<Key: store, Value: store>(map: SimpleMap<Key, Value>): vector<(Key, Value)> { .. }
```